core/src/main/java/org/apache/brooklyn/core/entity/internal/EntityConfigMap.java

```diff
@@ -132,7 +132,11 @@ public EntityConfigMap(AbstractEntity entity, Map<ConfigKey<?>, Object> storage)
 
         // Get own value
         if (((ConfigKeySelfExtracting<T>)key).isSet(ownConfig)) {
-            ownValue = Maybe.of(((ConfigKeySelfExtracting<T>)key).extractValue(ownConfig, exec));
+            Map<ConfigKey<?>, ?> ownCopy = null;
```

Comment:
Very minor: personal preference to not set its value to null when we're about to set its value immediately below that. Let the compiler tell us if someone changing it makes a mistake so the value no longer gets initialised.